src/common/utils/useForm.tsx

```diff
@@ -5,12 +5,14 @@ interface IValues {
   name: string;
   email: string;
   message: string;
+  phoneNumber: string;
 }
 

```

Comment:
- **Pre-condition check**  
  The `validate` function is not checked for null or undefined before being used. This could lead to a runtime error if `validate` is null or undefined.